### ISSUE TITLE
Some issue faced on using Slack on ferdi

### ISSUE THREAD

**Issue by JustCarousell** [getferdi/ferdi#2449]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/getferdi/ferdi/issues) for a bug report similar to mine, in vain

### Steps to reproduce

First Issue
1. Go to Slack
2. Click on any conversation and try to download maybe a file from someone
3. Click on download
4. Take me out of ferdi
5. Bring me to my browser and open up slack workspace login page

Second Issue
1. Go to Slack
2. Start a huddle
3. Get the person on the other end to share screen
4. On our end click on view screen
5. Not able to open up view screen in ferdi

### Debug link

https://debug.getferdi.com/6a42e259-191a-47e8-9be4-7863f1dc2599

### Additional information

So basically the two issue faced here is not able to download files and open up someone else shared screen natively in ferdi, not sure is it supposed to be the way it is or it's supposed to actually work. Any feedback, help or follow up from the community or the devs would be greatly appreciated, thank you!
